src/components/Select/components/Listbox/GroupLabel.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import {block} from '../../../utils/cn';
+import {SelectProps} from '../../types';
+
+const b = block('select-listbox');
+
+type GroupLabelProps = {
+    size: NonNullable<SelectProps['size']>;
+    label: string;
+};
+
+export const GroupLabel = ({size, label}: GroupLabelProps) => {
+    return <div className={b('group-label', {size})}>{label}</div>;
```

Comment:
Is it worth extracting a "oneliner"?

Reply:
I fixed text collapse. Now it looks like
```typescript
export const GroupLabel = ({label}: GroupLabelProps) => {
    return (
        <div className={b('group-label')}>
            <div className={b('group-label-content')}>{label}</div>
        </div>
    );
};
```